test/unit/android-helper-specs.js

```diff
@@ -102,10 +102,20 @@ describe('Android Helpers', () => {
   });
   describe('ensureDeviceLocale', withMocks({adb}, (mocks) => {
     it('should call setDeviceLanguageCountry', async () => {
-      mocks.adb.expects('setDeviceLanguageCountry').withExactArgs('en', 'US').once().returns("");
+      mocks.adb.expects('setDeviceLanguageCountry').withExactArgs('en', 'US').once();
       await helpers.ensureDeviceLocale(adb, 'en', 'US');
       mocks.adb.verify();
     });
+    it('should call setDeviceLanguageCountry with undefined', async () => {
+      mocks.adb.expects('setDeviceLanguageCountry').withExactArgs(undefined, undefined).once();
```

Comment:
I meant we could probably check that it throws an exception if an invalid language/country name is provided. Otherwise these tests literally verify the same thing as the previous one and are redundant.

Reply:
ah, understood.

`Locale` method in [io.appium.settings's PR](https://github.com/appium/io.appium.settings/pull/11#discussion_r150606119) doesn't raise exceptions for any letters, and `setDeviceLanguageCountry` also doesn't throw exceptions.

Variation for arguments, no arguments and one argument for `setDeviceLanguageCountry`, are tested in https://github.com/appium/appium-adb/pull/271/files .

So, in this PR, whether this method can call `setDeviceLanguageCountry` or not is enough for test code, I thought.